### ISSUE TITLE
special handling for .RData files

### ISSUE THREAD

**Issue by williamstein** [sagemathinc/cocalc#7409]
If a user clicks to open an RData file, they are repeatedly warned not to proceed or it will be corrupted.  However, sometimes they do anyways.  Instead, we should:

1. Display a message about what an RData file is
2. Suggest running R with a link to our docs
3. Explain how to load an RData file, e.g., linking to https://bookdown.org/ndphillips/YaRrr/rdata-files.html

**Comment by novoselt** [comment]
In general, if it is an unsupported file type, perhaps the only option can be to display it in some read-only way, so that the actual file on disk is intact?

**Comment by williamstein** [comment]
Yes, but in this case there could be many useful things to say.  It might even be possible to run a 1-line R command on the RData file and get a summary of the objects in it.
